linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -225,15 +228,21 @@ private[emitter] final class ClassEmitter(jsGen: JSGen) {
       WithGlobals(js.Skip())
     } { parentIdent =>
       val (inheritedCtorDefWithGlobals, inheritedCtorRef) = if (!isJSClass) {
-        (WithGlobals(js.Skip()), envField("h", parentIdent.name))
+        if (className == ThrowableClass) {
+          val inheritableCtorDef =
+            makeInheritableCtorDef(js.VarRef(js.Ident("Error")), "hh")
+          (WithGlobals(inheritableCtorDef), envField("hh", className))
```

Comment:
I don't understand why we need this machinery (rather than just assigning `new Error()` to `prototyp`). Then again, `Error` seems pretty magic :S

Reply:
We always do that machinery. It is necessary so that the `prototype` does not have the *fields* of the superclass. It must only chain to the prototype of the parent class, but not introduce any field. There's nothing specific to `Error` here.

Reply:
Yeah, that's fair. It's just a bit ugly that we need to introduce `hh`. But OK :)